### PR TITLE
fix document typo in tf.Module

### PR DESCRIPTION
**Pull request by JiaRu2016**
to close issue #37293

### CODE REVIEW

**Comment by JiaRu2016** [comment]
@googlebot I signed it!

**Comment by mihaimaruseac** [comment]
I'm not sure the new argument names are valid either. Here's is the relevant part of the `Dense` constructor: https://github.com/tensorflow/tensorflow/blob/3820a4ac5d017b5c8ea179b2c5b99d7b89004f02/tensorflow/python/keras/layers/core.py#L1116-L1132

and here is the relevant part of the `Layer` (the superclass of `Dense`) constructor: https://github.com/tensorflow/tensorflow/blob/3820a4ac5d017b5c8ea179b2c5b99d7b89004f02/tensorflow/python/keras/engine/base_layer.py#L278-L293

So, `in_features` and `out_features` are neither arguments of either of the two `__init__`s nor are they members of the `allowed_kwargs`

**Comment by gbaned** [comment]
@JiaRu2016 Can you please check @mihaimaruseac's comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
